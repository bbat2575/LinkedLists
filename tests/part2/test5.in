NEW 1
1
DELETE 0 1
DELETE 1 0
DELETE -1 0
DELETE 0 -2
DELETE 0 0 0
DELETE  0 0
DELETE 0 a
DELETE a 0
DELETE 999 0
DELETE -999 0
DELETE 0 999
DELETE 0 -999
DELETE 0.0 0
DELETE 0 0.0
DELETE 0. 0
DELETE 0 0.
DELETE .0 0
DELETE 0 .0
DELETE 0e0 0
DELETE 0 0e0
DELETE 2e-6 0
DELETE 0 2e-6
DELETE 00
DELETE 0
DELETE
DELETE  
DELETE -0 0
DELETE +0 0
DELETE - 0
DELETE + 0
DELETE 0 -
DELETE 0 +
DELETE00
DELETE0
DELETE0 0
  DELETE
0 0 DELETE
  DELETE 0 0 2
DELET E 0 0
DEL ETE 0 0
D ELETE 0 0
delete 0 0
Delete 0 0
DELETe 0 0
EDLETE 0 0
