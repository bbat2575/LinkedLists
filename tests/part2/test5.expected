
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: DELETE

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
